WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -303,9 +294,14 @@ private async Task HandleCoinJoinFinalizationAsync(CoinJoinTracker finishedCoinJ
 			Logger.LogError($"{logPrefix} failed with exception:", e);
 		}
 
-		foreach (var coins in finishedCoinJoin.CoinCandidates)
+		if (!trackedCoinJoins.TryRemove(walletToRemove.WalletName, out _))
```

Comment:
This code part was above as it is basically here I just switch the order. Hard to see in diff - no code change.